.github/workflows/test-sov.yml

```
@@ -0,0 +1,48 @@
+name: Run PR test
+
+on:
+  push:
```

what I meant here was to do 
```suggestion
  push:
      branches: [ main ]
  pull_request:
```